content/docs/handling-events.md

```diff
@@ -60,8 +60,10 @@ When using React, you generally don't need to call `addEventListener` to add lis
 
 When you define a component using an [ES6 class](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Classes), a common pattern is for an event handler to be a method on the class. For example, this `Toggle` component renders a button that lets the user toggle between "ON" and "OFF" states:
 
-```js{6,7,10-14,18}
-class Toggle extends React.Component {
+```js codesandbox=cra?highlights=8,9,12,13,14,15,16,20&height=400
```

Comment:
🤯

Reply:
Cool right?! Would be even nicer if we can update the line highlight setting to copy the exact highlights!